packages/nuxt/src/index.ts

```diff
@@ -25,6 +27,7 @@ export default defineNuxtModule<UnocssNuxtOptions>({
     attributify: false,
     webFonts: false,
     icons: false,
+    inspector: false,
```

Comment:
It's true by default in Vite plugin, any reason to have it default to false? https://github.com/unocss/unocss/blob/c1aa6d03b9c4cd7124582348df0e42d48639da66/packages/vite/src/types.ts#L7